scripts/regenerate_runtimes.sh

```diff
@@ -3,5 +3,8 @@
 cd tools/runtime-codegen
 cargo run --bin runtime-codegen -- --from-node-url "wss://rococo-bridge-hub-rpc.polkadot.io:443" > ../../relays/client-bridge-hub-rococo/src/codegen_runtime.rs
 cargo run --bin runtime-codegen -- --from-node-url "http://localhost:20433" > ../../relays/client-rialto-parachain/src/codegen_runtime.rs
+cargo run --bin runtime-codegen -- --from-node-url "wss://rococo-rpc.polkadot.io:443" > ../../relays/client-rococo/src/codegen_runtime.rs
```

Comment:
I guess we're fine to use Rococo runtime api for Wococo too.. but taking mental note to look at this if in the future things start failing on Wococo all of a sudden 😆